.github/workflows/ci.yml

```
@@ -14,8 +14,8 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04]
-        python-version: ['3.8']
-        toxenv: [quality, django32, django42, check_keywords]
+        python-version: ['3.8', '3.11', '3.12']
```

Since we are upgrading to `Python 3.11` now, we can skip running tests with `3.12` for now.
```suggestion
        python-version: ['3.8', '3.11']
```

---

I think since the tests are passing on both, we should run both, we'll want to drop 3.11 after we cut redwood almost immediately so best to start testing 3.12 where it's already passing.